tiny_dnn/util/serialization_functions.h

```diff
@@ -242,6 +242,90 @@ struct LoadAndConstruct<tiny_dnn::slice_layer> {
   }
 };
 
+template <>
+struct LoadAndConstruct<tiny_dnn::sigmoid_layer> {
+  template <class Archive>
+  static void load_and_construct(
```

Comment:
Please read
https://github.com/tiny-dnn/tiny-dnn/blob/master/cereal/access.hpp#L42
to understand why `LoadAndConstruct<serialized_type>::load_construct` methods need to be writen.